### PR TITLE
Single Line CLI Output

### LINKED ISSUE #1148: Ensure a single line of CLI output per query
## Steps To Reproduce

Create a page with multiple queries and build it

## Expected Behavior

A single line of CLI output per query

## Actual Behaviour

Due to changes made when we added multithreading, the query puts print on top of each other

CleanShot 2023-09-05 at 13 06 54@2x

### PR DESCRIPTION
**Pull request by ItsMeBrianD**
### Description

Executing queries now only logs a single line; this makes the output much more legible.
The filename was unable to be added in a straightforward way; as the endpoint that runs queries only receives the route hash.

image

Resolves #1148 

### Checklist

- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### CODE REVIEW

File: packages/db-orchestrator/index.cjs [hunk 1]
```diff
@@ -177,8 +176,8 @@ const runQueries = async function (routeHash, dev) {
 							populateColumnTypeMetadata(data, queryIndex, columnTypes);
 
 							readline.cursorTo(process.stdout, 0);
```
Comment (csjh) [inline packages/db-orchestrator/index.cjs:178]: Are these still necessary?

Comment (csjh) [inline packages/db-orchestrator/index.cjs:178]: Are the `readline.cursorTo(process.stdout, 0);`s still necessary?

Comment (ItsMeBrianD) [reply packages/db-orchestrator/index.cjs:178]: Ah likely not

**Comment by mcrascal** [comment]
LGTM

**Review by csjh — APPROVED** [review]

**Comment by archiewood** [comment]
This netlify preview has built sucessfully, just for some reason the check status has not updated 

https://deploy-preview-1165--evidence-development-workspace.netlify.app/

**Comment by archiewood** [comment]
@ItsMeBrianD want to merge?

**Pull request merged**